.pre-commit-config.yaml

```diff
@@ -15,34 +16,31 @@ repos:
       - id: detect-private-key
       - id: check-merge-conflict
 
-  - repo: https://github.com/pre-commit/mirrors-jshint
-    rev: v2.13.6
+  - repo: local
     hooks:
-      - id: jshint
-        files: "(^.*\\.js$)"
+      - id: coding-standard-fixer
+        name: Check the PHP and SFCC coding standards for JS and autofix
+        language: system
+        entry:  npx sgmf-scripts --lint js --fix
+        stages: [commit]
 
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v8.29.0
-    hooks:
-      - id: eslint
-        files: "(^cartridges/int_alma/cartridge/.*\\.js$)"# *.js, *.jsx, *.ts and *.tsx
-        types: [file]
-        args:
-          - --fix
+      - id: coding-standard-fixer
+        name: Check the PHP and SFCC coding standards for CSS and autofix
+        language: system
+        entry:  npx sgmf-scripts --lint css --fix
+        stages: [commit]
 
-  - repo: https://github.com/thibaudcolas/pre-commit-stylelint
-    rev: v14.16.0
-    hooks:
-      - id: stylelint
-        files: "(^cartridges/int_alma/cartridge/static/default/css/.*\\.css$)"
-        args:
-          - --config
-          - .stylelintrc.json
-        additional_dependencies:
-          - stylelint@15.10.2
-          - prettier@2.8.8
-          - stylelint-config-standard@32.0.0
-          - stylelint-prettier@3.0.0
+      - id: coding-standard
+        name: Check the PHP and SFCC coding standards for JS
+        language: system
+        entry:  npx sgmf-scripts --lint js
+        stages: [commit]
+
+      - id: coding-standard
+        name: Check the PHP and SFCC coding standards for CSS
+        language: system
+        entry:  npx sgmf-scripts --lint css
+        stages: [commit]
```

Comment:
No need to duplicate these hooks, the `--fix` will show and fix issues already. 
We can remove those.

Reply:
The idea was to show the issues remaining after the fix. 
I think that one of the linter was not showing them so I replicated this behaviour everywhere, but I checked with this one and indeed the `--fix` shows the errors remaining, so yes it's redundant, I'm removing them
(And maybe I should double check for the other repositories :/ )